base_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './nameable'
```

Comment:
### Optional
- [ ] It would be nice if we use `require_relative` when working with files in our directories, we use `require` when working with packages/gems in our gemfiles